CONTRIBUTING.md

```diff
@@ -68,6 +68,9 @@ It allows us to do following when saving a file:
 * Remove unnecessary semicolon
 * Add missing @Override annotations
 
+Configure Intellij to [insert a new line at the end of a file](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file)
```

Comment:
Could you add a heading such as "New line at the end of a file"?

Reply:
Sure, I will add the heading.